src/sentry_envelope.c

```diff
@@ -200,15 +200,23 @@ sentry_envelope_get_event(const sentry_envelope_t *envelope)
         return sentry_value_new_null();
     }
     for (size_t i = 0; i < envelope->contents.items.item_count; i++) {
+
+#ifdef SENTRY_PERFORMANCE_MONITORING
```

Comment:
now that I see this again, maybe we *should* return the transaction here. remember: a transaction *is* an event, just a very special one. It has an id, and can have associated attachments.

Reply:
i'm going to avoid referencing more than one SDK here, but [python](https://github.com/getsentry/sentry-python/blob/49cae6009a4e39c47ef8834b07668f5eb9789ca8/sentry_sdk/envelope.py#L256-L269) for example deliberately has separate getters for an event and a transaction in an envelope, for a counterexample.